src/main/java/io/neonbee/handler/ErrorHandler.java

```diff
@@ -0,0 +1,21 @@
+package io.neonbee.handler;
+
+import io.neonbee.NeonBee;
+import io.vertx.core.Future;
+
+/**
+ * The class which implements this interface MUST have a default constructor. Otherwise NeonBee can't instantiate the
+ * class during bootstrap phase.
+ */
+@SuppressWarnings("NM_SAME_SIMPLE_NAME_AS_INTERFACE")
+public interface ErrorHandler extends io.vertx.ext.web.handler.ErrorHandler {
+
```

Comment:
Can we add a rule to not have this "initial blank line" in all classes? It drives me nuts! :D Thanks.